ClinGen/clincoded
# User and Affiliation Updates for Release 27

Steps to test #1895:
1. Choose to curate as the **Monogenic Diabetes** affiliation.
2. Via the GCI's approval process, save, provision, approve and publish a gene-disease classification.

Steps to test #1940:
1. Choose to curate as the **Peroxisomal Disorders** affiliation (to confirm it now exists).

Steps to test #1951:
1. Choose to curate as the **Coagulation Factor Deficiency** affiliation (to confirm the new name).
